src/applications/static-pages/facilities/FacilityMapWidget.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { apiRequest } from '../../../platform/utilities/api';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+import FacilityApiAlert from './FacilityApiAlert';
+import { mapboxToken } from '../../facility-locator/components/MapboxClient';
+import { buildAddressArray } from '../../facility-locator/utils/facilityAddress';
+import environment from '../../../platform/utilities/environment';
+
+export default class FacilityDetailWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+    };
+  }
+
+  componentDidMount() {
```

Comment:
we should probably use the redux store from now on, especially since this is on the detail page only, right?

Reply:
this will also make unit testing a ton easier